cmake/KokkosCore_config.h.in

```diff
@@ -78,6 +78,7 @@
 #cmakedefine KOKKOS_ARCH_POWER7
 #cmakedefine KOKKOS_ARCH_POWER8
 #cmakedefine KOKKOS_ARCH_POWER9
+#cmakedefine KOKKOS_ARCH_RISCV_MILKV
```

Comment:
You may want to call this "SOPHON" or  "SG2042" or something more specific to the system CPU rather than the Milk-V platform, which is independent of the processor

Reply:
I moved to SG2042.